x/README.md

```diff
@@ -6,17 +6,18 @@ sidebar_position: 0
 
 Modules are the code components of the Provenance Blockchain that execute the majority of the business logic for applications. The [Cosmos SDK](https://docs.cosmos.network/v0.47) enables developers to build modules that utilize the core structure of the SDK to allow the modules to function together. To read more about creating modules, refer to the [Cosmos documentation on modules](https://docs.cosmos.network/v0.47/building-modules/intro).
 
-Provenance uses inherited modules from the Cosmos SDK, and has also developed modules that are specific to Provenance.
+Provenance Blockchain leveraged inherited modules from Cosmos SDK, and has purpose-built custom modules unique to Provenance Blockchain.
```

Comment:
I think you mean "leverages".

Reply:
I'll revert to marketing as this is their verbiage. I'll check with them and see if they'd like to make that change.

Reply:
Documentation usually isn't past tense. And the other stuff in this file is present tense, so the past tense here is probably not correct. I imagine the important part of the marketing verbiage is the inclusion of the word "Blockchain" after "Provenance".